### ISSUE TITLE
[Bug]:  crash parachute

### ISSUE THREAD

**Issue by binded-zz** [Yimura/YimMenu#1120]
### Describe the bug

its happened a few times now first time i was able to get the dump 
i hope its the right one as the time listed is not the right time as in the log

happens when i destroy some people flying and they open a chute idk what one it is and i dont think its all of the chutes doing it 

### Steps To Reproduce

idk happens  when people open parachutes idk what ones it is as it crashes right to the desktop hopefully you can figure it out from the dump

### Expected Behavior

not crash i thought there was a protections for it

### Screenshots and/or Logs

```
[14:15:22.5806321][ERROR/exception_handler.cpp:27] UNKNOWN_EXCEPTION
Dumping modules:
GTA5.exe Base Address: 0x7FF7C79A0000 Size: 63526912
ntdll.dll Base Address: 0x7FF8FFB50000 Size: 1245184
KERNEL32.DLL Base Address: 0x7FF8FE640000 Size: 528384
KERNELBASE.dll Base Address: 0x7FF8FD1D0000 Size: 1609728
apphelp.dll Base Address: 0x7FF8FA0C0000 Size: 335872
AcGenral.DLL Base Address: 0x7FF82A800000 Size: 253952
msvcrt.dll Base Address: 0x7FF8FE040000 Size: 507904
sechost.dll Base Address: 0x7FF8FEE40000 Size: 446464
SHLWAPI.dll Base Address: 0x7FF8FF000000 Size: 212992
USER32.dll Base Address: 0x7FF8FD960000 Size: 606208
win32u.dll Base Address: 0x7FF8FD570000 Size: 49152
GDI32.dll Base Address: 0x7FF8FED70000 Size: 57344
gdi32full.dll Base Address: 0x7FF8FCF90000 Size: 659456
msvcp_win.dll Base Address: 0x7FF8FD8C0000 Size: 335872
ucrtbase.dll Base Address: 0x7FF8FD0B0000 Size: 798720
ole32.dll Base Address: 0x7FF8FF060000 Size: 897024
combase.dll Base Address: 0x7FF8FDC00000 Size: 2539520
RPCRT4.dll Base Address: 0x7FF8FE0F0000 Size: 876544
advapi32.dll Base Address: 0x7FF8FDF90000 Size: 425984
shcore.dll Base Address: 0x7FF8FEBE0000 Size: 679936
SHELL32.dll Base Address: 0x7FF8FF320000 Size: 6492160
USERENV.dll Base Address: 0x7FF8FC640000 Size: 90112
MPR.dll Base Address: 0x7FF8E61E0000 Size: 73728
SspiCli.dll Base Address: 0x7FF8FC320000 Size: 126976
IMM32.DLL Base Address: 0x7FF8FE720000 Size: 126976
PSAPI.DLL Base Address: 0x7FF8FDBF0000 Size: 4096
WS2_32.dll Base Address: 0x7FF8FE5A0000 Size: 290816
VERSION.dll Base Address: 0x7FF8F60E0000 Size: 12288
GFSDK_ShadowLib.win64.dll Base Address: 0x7FF8034D0000 Size: 96256
DSOUND.dll Base Address: 0x7FF8292B0000 Size: 491520
CRYPT32.dll Base Address: 0x7FF8FD5A0000 Size: 1134592
WINTRUST.dll Base Address: 0x7FF8FD850000 Size: 307200
bink2w64.dll Base Address: 0x7FF8A58C0000 Size: 307712
WININET.dll Base Address: 0x7FF8E5920000 Size: 2113536
OLEAUT32.dll Base Address: 0x7FF8FDB10000 Size: 647168
WINMM.dll Base Address: 0x7FF8F5C80000 Size: 114688
MF.dll Base Address: 0x7FF8F6850000 Size: 368640
MFPlat.DLL Base Address: 0x7FF8F6620000 Size: 1503232
msdmo.dll Base Address: 0x7FF829470000 Size: 16384
MFReadWrite.dll Base Address: 0x7FF8298B0000 Size: 1003520
PROPSYS.dll Base Address: 0x7FF8F8230000 Size: 688128
ResampleDmo.DLL Base Address: 0x7FF8291F0000 Size: 503808
powrprof.dll Base Address: 0x7FF8FBEA0000 Size: 77824
winmmbase.dll Base Address: 0x7FF8E5E20000 Size: 102400
d3d9.dll Base Address: 0x7FF8D01C0000 Size: 1306624
D3DCOMPILER_43.dll Base Address: 0x7FF83F150000 Size: 2423296
WTSAPI32.dll Base Address: 0x7FF8FA160000 Size: 40960
GFSDK_TXAA_AlphaResolve.win64.dll Base Address: 0x7FF8DE6D0000 Size: 48128
DINPUT8.dll Base Address: 0x7FF8B3DB0000 Size: 176128
XINPUT1_3.dll Base Address: 0x400000 Size: 85504
SETUPAPI.dll Base Address: 0x7FF8FE760000 Size: 913408
bcrypt.dll Base Address: 0x7FF8FC940000 Size: 110592
IPHLPAPI.DLL Base Address: 0x7FF8FBB40000 Size: 122880
d3dx9_43.dll Base Address: 0x7FF803260000 Size: 2280960
cfgmgr32.dll Base Address: 0x7FF8FCC60000 Size: 221184
dwmapi.dll Base Address: 0x7FF8FA510000 Size: 90112
OPENGL32.dll Base Address: 0x7FF8C5F70000 Size: 798720
MFCORE.DLL Base Address: 0x7FF8294C0000 Size: 3252224
dxcore.dll Base Address: 0x7FF8FA3B0000 Size: 131072
GLU32.dll Base Address: 0x7FF8C5F00000 Size: 126976
UMPDC.dll Base Address: 0x7FF8FBE80000 Size: 36864
kernel.appcore.dll Base Address: 0x7FF8FC0D0000 Size: 36864
bcryptPrimitives.dll Base Address: 0x7FF8FD7D0000 Size: 380928
RTWorkQ.DLL Base Address: 0x7FF8F65E0000 Size: 102400
MSASN1.dll Base Address: 0x7FF8FCCB0000 Size: 36864
inputhost.dll Base Address: 0x7FF8E8C70000 Size: 1515520
CoreMessaging.dll Base Address: 0x7FF8F9DE0000 Size: 860160
CRYPTBASE.DLL Base Address: 0x7FF8FC7C0000 Size: 12288
sdk_legacy_led_x64.dll Base Address: 0x7FF802E50000 Size: 2625536
MSVCP140.dll Base Address: 0x7FF8E63B0000 Size: 344576
VCRUNTIME140.dll Base Address: 0x7FF8E6450000 Size: 64000
VCRUNTIME140_1.dll Base Address: 0x7FF8E6440000 Size: 14848
CRYPTSP.dll Base Address: 0x7FF8FC7A0000 Size: 57344
rsaenh.dll Base Address: 0x7FF8FC040000 Size: 147456
mswsock.dll Base Address: 0x7FF8FC540000 Size: 331776
windows.storage.dll Base Address: 0x7FF8FAFB0000 Size: 6909952
wintypes.dll Base Address: 0x7FF8FAE60000 Size: 544768
dbghelp.dll Base Address: 0x7FF8FAA50000 Size: 1626112
dbgcore.DLL Base Address: 0x7FF8F4E10000 Size: 131072
uxtheme.dll Base Address: 0x7FF8FA2A0000 Size: 425984
AudioDevProps2.dll Base Address: 0x7FF8BC200000 Size: 624640
clbcatq.dll Base Address: 0x7FF8FEF50000 Size: 471040
amsi.dll Base Address: 0x7FF8F1870000 Size: 57344
profapi.dll Base Address: 0x7FF8FCEC0000 Size: 73728
MpOav.dll Base Address: 0x7FF8F17F0000 Size: 204800
cryptnet.dll Base Address: 0x7FF8F5590000 Size: 139264
drvstore.dll Base Address: 0x7FF8F5430000 Size: 1044480
devobj.dll Base Address: 0x7FF8FCC30000 Size: 110592
wldp.dll Base Address: 0x7FF8FC850000 Size: 184320
imagehlp.dll Base Address: 0x7FF8FE620000 Size: 49152
D3D11.DLL Base Address: 0x7FF8F8F50000 Size: 1867776
dxgi.dll Base Address: 0x7FF8FA3F0000 Size: 651264
directxdatabasehelper.dll Base Address: 0x7FF8F0F50000 Size: 196608
nvapi64.dll Base Address: 0x7FF8F1EA0000 Size: 6144512
D3D10_1.DLL Base Address: 0x7FF8C5F30000 Size: 147456
d3d10_1core.dll Base Address: 0x7FF8D1C90000 Size: 12288
MSCTF.dll Base Address: 0x7FF8FF200000 Size: 917504
textinputframework.dll Base Address: 0x7FF8E9850000 Size: 925696
CoreUIComponents.dll Base Address: 0x7FF8F0950000 Size: 2036736
WINSTA.dll Base Address: 0x7FF8FBAC0000 Size: 229376
nvldumdx.dll Base Address: 0x7FF8EF920000 Size: 599552
nvwgf2umx.dll Base Address: 0x7FF8EBA90000 Size: 33566208
dcomp.dll Base Address: 0x7FF8F9780000 Size: 1720320
HID.DLL Base Address: 0x7FF8FB880000 Size: 24576
MMDevApi.dll Base Address: 0x7FF8F4980000 Size: 462848
AudioSes.dll Base Address: 0x7FF8E1A10000 Size: 1384448
ProductInfo.dll Base Address: 0x7FF8E9980000 Size: 56320
napinsp.dll Base Address: 0x7FF8E4320000 Size: 53248
pnrpnsp.dll Base Address: 0x7FF8E4300000 Size: 69632
DNSAPI.dll Base Address: 0x7FF8FBB70000 Size: 716800
NSI.dll Base Address: 0x7FF8FEDA0000 Size: 8192
winrnr.dll Base Address: 0x7FF8E4150000 Size: 32768
wshbth.dll Base Address: 0x7FF8F5EB0000 Size: 45056
nlansp_c.dll Base Address: 0x7FF8E4060000 Size: 81920
XAudio2_7.dll Base Address: 0x7FF89D630000 Size: 485376
resourcepolicyclient.dll Base Address: 0x7FF8FA810000 Size: 36864
Windows.UI.dll Base Address: 0x7FF8EB3F0000 Size: 966656
avrt.dll Base Address: 0x7FF8F6CB0000 Size: 8192
fwpuclnt.dll Base Address: 0x7FF8F5A30000 Size: 294912
rasadhlp.dll Base Address: 0x7FF8F60D0000 Size: 8192
gpapi.dll Base Address: 0x7FF8FC5D0000 Size: 77824
WINNSI.DLL Base Address: 0x7FF8FA260000 Size: 16384
socialclub.dll Base Address: 0x7FF805770000 Size: 5077504
chrome_elf.dll Base Address: 0x7FF8A5640000 Size: 685568
libcef.dll Base Address: 0x7FFFF7F50000 Size: 109755392
COMDLG32.dll Base Address: 0x7FF8FE4A0000 Size: 774144
OLEACC.dll Base Address: 0x7FF8C9950000 Size: 274432
UIAutomationCore.DLL Base Address: 0x7FF8B7CD0000 Size: 3047424
Secur32.dll Base Address: 0x7FF8E11B0000 Size: 12288
COMCTL32.dll Base Address: 0x7FF8D5200000 Size: 573440
NETAPI32.dll Base Address: 0x7FF8E61B0000 Size: 49152
WINSPOOL.DRV Base Address: 0x7FF8D52F0000 Size: 360448
DWrite.dll Base Address: 0x7FF8F8CE0000 Size: 1605632
USP10.dll Base Address: 0x7FF87F830000 Size: 4096
credui.dll Base Address: 0x7FF8F8720000 Size: 65536
dxva2.dll Base Address: 0x7FF89DDF0000 Size: 102400
WINHTTP.dll Base Address: 0x7FF8F6A80000 Size: 958464
urlmon.dll Base Address: 0x7FF8F1480000 Size: 1298432
ncrypt.dll Base Address: 0x7FF8FC910000 Size: 114688
CRYPTUI.dll Base Address: 0x7FF8F4470000 Size: 278528
dhcpcsvc.DLL Base Address: 0x7FF8F6380000 Size: 69632
wevtapi.dll Base Address: 0x7FF8F6070000 Size: 245760
ESENT.dll Base Address: 0x7FF8D5830000 Size: 3112960
iertutil.dll Base Address: 0x7FF8F0FA0000 Size: 557056
srvcli.dll Base Address: 0x7FF8F1260000 Size: 57344
netutils.dll Base Address: 0x7FF8FBB30000 Size: 20480
WKSCLI.DLL Base Address: 0x7FF8F5FF0000 Size: 49152
SAMCLI.DLL Base Address: 0x7FF8F47C0000 Size: 61440
NTASN1.dll Base Address: 0x7FF8FC8D0000 Size: 69632
FirewallAPI.dll Base Address: 0x7FF8FB9B0000 Size: 380928
fwbase.dll Base Address: 0x7FF8FB970000 Size: 163840
FWPolicyIOMgr.dll Base Address: 0x7FF8E6010000 Size: 241664
wdmaud.drv Base Address: 0x7FF88CE00000 Size: 200704
ksuser.dll Base Address: 0x7FF8D2EF0000 Size: 8192
msacm32.drv Base Address: 0x7FF8D1A70000 Size: 20480
MSACM32.dll Base Address: 0x7FF88E490000 Size: 77824
midimap.dll Base Address: 0x7FF8CE540000 Size: 16384
ondemandconnroutehelper.dll Base Address: 0x7FF8B0F70000 Size: 45056
YimMenu.dll Base Address: 0x7FF802A40000 Size: 2357760
D3DCOMPILER_47.dll Base Address: 0x7FF8F8740000 Size: 3043328
schannel.DLL Base Address: 0x7FF8FBF40000 Size: 544768
ncryptsslp.dll Base Address: 0x7FF8DBB20000 Size: 86016
xinput1_4.dll Base Address: 0x7FF8F6940000 Size: 32768
deviceaccess.dll Base Address: 0x7FF8D1B40000 Size: 151552
dhcpcsvc6.DLL Base Address: 0x7FF8F6CC0000 Size: 28672
wbemprox.dll Base Address: 0x7FF8F47E0000 Size: 28672
wbemcomn.dll Base Address: 0x7FF8F1E20000 Size: 311296
wbemsvc.dll Base Address: 0x7FF8F3360000 Size: 8192
fastprox.dll Base Address: 0x7FF8F1980000 Size: 708608
Dumping registers:
RAX: 0x7FF8F8F72612
RCX: 0x0
RDX: 0xDEB6B7EB80
RBX: 0x7FF8F9172B40
RSI: 0xDEB6B7F528
RDI: 0x19930520
RSP: 0xDEB6B7F220
RBP: 0xDEB6B7F729
R8:  0x7FF8F8F81078
R9:  0x0
R10: 0x26BBA181D40
R11: 0x4B400000002
R12: 0x0
R13: 0x1
R14: 0x26BFA520150
R15: 0xDEB6B7F738
Dumping stacktrace:
[0]	KERNELBASE.dll RaiseException
[1]	ucrtbase.dll CxxThrowException
[2]	D3D11.DLL D3D11CreateDeviceAndSwapChain
[3]	D3D11.DLL EnableFeatureLevelUpgrade
[4]	GTA5.exe+0x1366B8B 0x7FF7C8D06B8B
[5]	GTA5.exe+0x1372AF0 0x7FF7C8D12AF0
[6]	GTA5.exe+0x136F410 0x7FF7C8D0F410
[7]	GTA5.exe+0x136DBF4 0x7FF7C8D0DBF4
[8]	GTA5.exe+0x136D847 0x7FF7C8D0D847
[9]	GTA5.exe+0x13645B9 0x7FF7C8D045B9
[10]	GTA5.exe+0x136C443 0x7FF7C8D0C443
[11]	GTA5.exe+0x9134F4 0x7FF7C82B34F4
[12]	GTA5.exe+0x915E2C 0x7FF7C82B5E2C
[13]	GTA5.exe+0x12E7AC1 0x7FF7C8C87AC1
[14]	GTA5.exe+0x12E981E 0x7FF7C8C8981E
[15]	KERNEL32.DLL BaseThreadInitThunk
[16]	ntdll.dll RtlUserThreadStart
--------End of exception--------

```

### Store

Rockstar

### Build

- [X] I've confirmed the issue exists on the latest version of YimMenu

**Comment by epicmusicpt** [comment]
currently im not having this issue anymore.

**Comment by binded-zz** [comment]
its still there i just crashed and there is no log it just ended  one of the random lines it always makes but cut off before the end 

its always random sometimes they pull it and nothing happens others times boom freeze with no ctd and have to force close the game or if lucky it does ctd

**Comment by epicmusicpt** [comment]
> its still there i just crashed and there is no log it just ended one of the random lines it always makes but cut off before the end
> 
> its always random sometimes they pull it and nothing happens others times boom freeze with no ctd and have to force close the game or if lucky it does ctd

check this might help,you can test: https://github.com/YimMenu/YimMenu/issues/1073

**Comment by binded-zz** [comment]
[16:08:35.1121426][INFO/gta_thread_start.cpp:13] Script Thread 'AM_MP_SMPL_INTERIOR_EXT' started.
[16:08:35.4538295][INFO/gta_thread_kill.cpp:11] Script Thread 'AM_MP_PROPERTY_EXT' terminated (via scrThread::Kill on myself).
[16:08:35.4544503][INFO/gta_thread_kill.cpp:11] Script Thread 'AM_MP_PROPERTY_EXT' terminated (via scrThread::Kill on myself).
[16:08:35.4910654][INFO/gta_thread_kill.cpp:11] Script Thread 'AM_MP_SMPL_INTERIOR_EXT' terminated (via scrThread::Kill on myself).
[16:08:35.4910880][INFO/gta_thread_kill.cpp:11] Script Thread 'AM_MP_SMPL_INTERIOR_EXT' terminated (via scrThread::Kill on myself).
[16:08:40.1948363][INFO/gta_thread_kill.cpp:11] Script Thread 'AM_MP_PROPERTY_EXT' terminated (via scrThread::Kill on myself).
[16:08:40.1954085][INFO/gta_thread_kill.cpp:11] Script Thread 'AM_MP_PROPERTY_EXT' terminated (via scrThread::Kill on myself).
[16:08:40.7731452][INFO/gta_thread_kill.cpp:11] Script Thread 'AM_MP_PROPERTY_EXT' terminated (via scrThread::Kill on myself).
[16:08:40.7731749][INFO/gta_thread_kill.cpp:11] Script Thread 'AM_MP_PROPERTY_EXT' terminated (via scrThread::Kill on myself).


just happened again froozen and then had to force close it 

 idk i just fly around in my heli and kill people flying around or what not if i out right kill them nothing happens but if they jump out or survive and fall and open near the ground  it freezes the radio keeps going just fine but every ting is locked 

i had thought maybe it had something to do with modestM being coloaded but it happens with out it as well

i guess its just my karma to get crashed every time i shot some out of the sky

**Comment by epicmusicpt** [comment]
did you try the fix by @inthecatsdreams? in https://github.com/YimMenu/YimMenu/issues/1073

**Comment by binded-zz** [comment]
> did you try the fix by @inthecatsdreams? in #1073

i dont build i use the night downloads

**Comment by epicmusicpt** [comment]
> > did you try the fix by @inthecatsdreams? in #1073
> 
> i dont build i use the night downloads

bro read the lasts posts carefully...i also dont build, @inthecatsdreams just did it for us,check the coments or go to his repo or fork. https://github.com/inthecatsdreams/25-Cobainen

**Comment by Yimura** [comment]
Most likely fixed.
